blockchain/package.json

```
@@ -29,21 +29,21 @@
     "*.json": "prettier --write",
     "*.sol": [
       "prettier --write --plugin=prettier-plugin-solidity",
-      "solhint --fix"
+      "solhint --fix --noPrompt 'contracts/**/*.sol'"
```

nit: what do you think?

```suggestion
      "npm run fix:prettier",
      "npm run fix:solhint"
```